app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -73,18 +73,22 @@
         line-height: 1.3;
       }
 
-      a {
-        text-decoration: none;
+      .contents-text,
+      .contents-number {
+        display: table-cell;
+      }
 
-        // Child spans are floated which creates a focus box around
-        // just the number. Set parent link to inline-block for
-        // correct appearance
-        display: inline-block;
+      .contents-number {
+        min-width: 1.3em;
```

Comment:
Increased in cc4f21d

![screen shot 2017-07-03 at 14 22 46](https://user-images.githubusercontent.com/319055/27794700-1c630bcc-5ffb-11e7-8ec5-2a46622f97ad.png)